### PR TITLE
Bookclub branch

### PR DESCRIPTION
**Pull request by j4sum**
## Type Of Change
- [x] Update documentation
- [ ] Add new event details
- [x] Add project resources

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] Commit message is as per guidelines in the contributor guide

### CODE REVIEW

**Comment by j4sum** [comment]
Once approved, I will update where required

File: resources/book-club/README.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+<td>Book<td>Author<td>Description<td>Blog
+<tr>
+<td><b>
+  <img src="https://m.media-amazon.com/images/I/61QZMnssyVL.jpg" />
+<td>
+The Grit Factor by Shannon Huffman Polson
+<td>
+What does it take for women to succeed in a male-dominated world? The Grit Factor. In writing The Grit Factor, Polson made it her mission to connect with an elite pack of tough, impressive female iconoclasts who shared with her their candid stories of combat and career. This slate of decorated leaders includes Heather Penney, one of the first female F-16 pilots, who was put on a suicide mission for 9/11; General Ann Dunwoody, the first female four-star general in the Army; Amy McGrath, the first female Marine to fly the F/A-18 in combat and a 2020 candidate for the US Senate—and dozens of other unstoppable women who got there first, including Polson herself.
+<td>
+ [Blogpost](https://iren-kamalova.medium.com/grit-factor-cc597dbf107f)
```
Comment (bharitwal) [inline resources/book-club/README.md:28]: This is not embedding the url correctly.

You can see it here : https://github.com/j4sum/WWCode-London/tree/bookclub-branch/resources/book-club

File: resources/book-club/README.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+<tr>
+<td><b>
+  <img src="https://cdn.waterstones.com/bookjackets/large/9780/1410/9780141029191.jpg" />
+<td>
+Quiet by Susan Cain                                     
+<td>
+In Quiet, Susan Cain argues that we dramatically undervalue introverts and shows how much we lose in doing so. She charts the rise of the Extrovert Ideal throughout the twentieth century and explores how deeply it has come to permeate our culture. She also introduces us to successful introverts—from a witty, high-octane public speaker who recharges in solitude after his talks, to a record-breaking salesman who quietly taps into the power of questions. Passionately argued, superbly researched, and filled with indelible stories of real people, Quiet has the power to permanently change how we see introverts and, equally important, how they see themselves. 
+<td>
+<tr>
```
Comment (bharitwal) [inline resources/book-club/README.md:68]: Blog Post can be added : https://www.womenwhocode.com/blog/quiet-book-club-wwcode-london

File: resources/book-club/README.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+</table>
+ 
+
+&nbsp;
+&nbsp;
+&nbsp;
+
+#### FOLLOW US ON
+
+- LinkedIn - https://www.linkedin.com/company/wwcodelondon/events/
```
Comment (bharitwal) [inline resources/book-club/README.md:90]:
```suggestion
- [LinkedIn](https://www.linkedin.com/company/wwcodelondon/events/)
```

File: resources/book-club/README.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+- Meetup - https://meetu.ps/c/1Cj0v/yRb71/d
```
Comment (bharitwal) [inline resources/book-club/README.md:92]:
```suggestion
- [Meetup](https://meetu.ps/c/1Cj0v/yRb71/d)
```

Comment (bharitwal) [reply resources/book-club/README.md:92]: Would be nice to add all our social media handles.

You can refer to the information here: https://github.com/WomenWhoCode/WWCode-London#social

**Review by bharitwal — CHANGES_REQUESTED** [review]
LGTM. 

Requested some minor changes.

**Comment by bharitwal** [comment]
fixes #27

**Comment by bharitwal** [comment]
Hey @j4sum - Since this PR has been open for a while it is out of sync with with the fork. Please can you sync the changes in your fork.

**Pull request closed without merging**